composer.json

```diff
@@ -7,8 +7,8 @@
         "guzzlehttp/guzzle":          "^7.0",
         "guzzlehttp/guzzle-services": "^1.0",
         "netresearch/jsonmapper":     "^1.4",
-        "symfony/options-resolver":   "^3.4|^4.4|^5.1|^6.0",
-        "psr/log": "^2.0|^3.0"
+        "symfony/options-resolver":   "^3.4|^4.4|^5.1|^6.0|^7.0",
+        "psr/log": "^1.1|^2.0|^3.0"
```

Comment:
need 1.X version because 2.0 and more need PHP 8.X
So if we not add 1.X version we can remove version 7.X of PHP support